### ISSUE TITLE
Handle x-ms-paths

### ISSUE THREAD

**Issue by salameer** [Azure/openapi-validation-tools#51]
- Currently the tool is blindly unifying x-ms-paths. This can be a problem if the paths are not unique. Make sure that the paths are unique.

**Comment by veronicagg** [comment]
@amarzavery is this still an issue?

**Comment by amarzavery** [comment]
yes. We are still blindly unifying x-ms-paths. This could be a linter rule to ensure that the union of paths and x-ms-paths is distinct. If this gets addressed before the PR is merged then we need not handle this here. It will be a linter rule in the SDK category.

**Comment by vladbarosan** [comment]
@amarzavery  is this task still relevant ? also you are saying that the union of paths and x-ms-paths is distinct. [isnt the requirement](https://github.com/Azure/autorest/wiki/xms-paths-in-path) for all x-ms-paths overload a path in the `paths` section ?
